source/extensions/filters/network/well_known_names.h

```diff
@@ -23,7 +23,7 @@ class NetworkFilterNameValues {
   // Mongo proxy filter
   const std::string MongoProxy = "envoy.mongo_proxy";
   // Rate limit filter
-  const std::string RateLimit = "envoy.ratelimit";
+  const std::string RateLimit = "envoy.rate_limit";
```

Comment:
I don't think we can change this. This will break config for people. I also don't think it's worth going through a deprecation cycle for this, though annoying. Can you check instead to make sure the docs are correct in various places?

Reply:
oh.. Yeah. Sorry. forgot about the deprecation cycle. I have reverted this and added docs that clarifies this.